src/items/rules/narrative_starship_combat_(optional).json

```diff
@@ -0,0 +1,352 @@
+{
+  "_id": "gZwp5BEskTN14fiy",
+  "name": "Narrative Starship Combat (OPTIONAL)",
+  "folder": "af3VzPTGTZOlzRqF",
+  "pages": [
+    {
+      "_id": "a6L3PLUMMA3tG4rX",
+      "name": "Introduction",
+      "type": "text",
+      "_stats": {
+        "coreVersion": "11.314",
+        "createdTime": 1699465298142,
+        "lastModifiedBy": "Ze895EIq7nm5kMrK",
+        "modifiedTime": 1699465402467,
+        "systemId": "sfrpg",
+        "systemVersion": "0.25.2"
+      },
+      "flags": {},
+      "image": {},
+      "ownership": {
+        "Ze895EIq7nm5kMrK": 3,
+        "default": -1
+      },
+      "sort": 100000,
+      "src": null,
+      "system": {},
+      "text": {
+        "content": "<p><strong>Source</strong>:  SFE, pp. 176-183</p><p>The starship combat system presented in the Starfinder Core Rulebook is a deep set of rules involving multiple dice rolls, hex-based measurement, and a wide array of choices for building and upgrading a spacefaring vessel. It’s the perfect fit for those groups who enjoy thinking tactically on a turn-by-turn basis and finding the “best” loadout of weapons and shields. Other players might want something a little more fast-paced that still allows them to make important decisions, roll some dice, and engage in memorable encounters.</p><p></p><p>This section introduces a more narrative-focused system for running starship combats. Rather than taking place on a hex grid, these combats rely more on “the theater of the mind,” focusing on cinematic interactions between the fighting starships. Every member of the crew still gets to participate in the encounter, lending their expertise to sway the outcome in their group’s favor!</p>",
```

Comment:
```suggestion
        "content": "<p><strong>Source</strong>: EN, pp. 176-183</p><p>The starship combat system presented in the Starfinder Core Rulebook is a deep set of rules involving multiple dice rolls, hex-based measurement, and a wide array of choices for building and upgrading a spacefaring vessel. It’s the perfect fit for those groups who enjoy thinking tactically on a turn-by-turn basis and finding the “best” loadout of weapons and shields. Other players might want something a little more fast-paced that still allows them to make important decisions, roll some dice, and engage in memorable encounters.</p><p></p><p>This section introduces a more narrative-focused system for running starship combats. Rather than taking place on a hex grid, these combats rely more on “the theater of the mind,” focusing on cinematic interactions between the fighting starships. Every member of the crew still gets to participate in the encounter, lending their expertise to sway the outcome in their group’s favor!</p>",
```